analyzer/codechecker_analyzer/analyzers/cppcheck/__init__.py

```diff
@@ -0,0 +1,5 @@
+# -------------------------------------------------------------------------
+#                     The CodeChecker Infrastructure
+#   This file is distributed under the University of Illinois Open Source
+#   License. See LICENSE.TXT for details.
+# -------------------------------------------------------------------------
```

Comment:
Why did the licencing change here to the old style? I thought CodeChecker moved to Apache 2.0 w/ LLVM, together with LLVM.

Reply:
Thanks, changed the licences.